.github/kokoro/db-full.sh

```diff
@@ -46,9 +46,9 @@ echo "----------------------------------------"
 	echo "----------------------------------------"
 
 	# Run the fuzzers
-	export MAX_VIVADO_PROCESS=$CORES
+	export MAX_VIVADO_PROCESS=$((CORES/2 < 20 ? CORES/2 : 20))
```

Comment:
Why are we limiting to 20? Please add a comment to explain why it is using a ceiling and the choice of 20.

Reply:
Done.